scanner/config/config.go

```diff
@@ -181,7 +185,9 @@ func (c *MatcherConfig) validate() error {
 			return fmt.Errorf("indexer_addr: failed to parse address: %w", err)
 		}
 	}
-
+	if _, err := url.Parse(c.VulnerabilitiesURL); err != nil {
```

Comment:
we should ensure it's not empty, as `url.Parse` allows the empty string: https://go.dev/play/p/do5J93glZR3

I will resolve this, as this will be resolved in a separate PR: https://github.com/stackrox/stackrox/pull/9417